pandaharvester/harvesterpreparator/pilotmover_preparator.py

```diff
@@ -70,13 +69,18 @@ def trigger_preparation(self, jobspec):
         else:
             return False, ErrMsg
 
+    #set local path for downloading
+    #def setBasePath(self, BasePath):
+    #    self.__basePath = BasePath
+    #    return self.__basePath
+
     # resolve input file paths
     def resolve_input_paths(self, jobspec):
         # get input files
         inFiles = jobspec.get_input_file_attributes()
         # set path to each file
         for inLFN, inFile in inFiles.iteritems():
-            inFile['path'] = mover_utils.construct_file_path(self.basePath, inFile['dataset'], inFile['scope'], inLFN)
+            inFile['path'] = self.basePath
```

Comment:
The base dir name only is part of the top of the storage tree. construct_file_path creates the file location account the rucio path convention. This is why the original code used the construct_file_path.